bots/faq_bot.py

```diff
@@ -30,11 +45,14 @@ async def on_message_activity(self, turn_context: TurnContext):
         conversation_data.channel_id = turn_context.activity.channel_id
         conversation_data.recipient_id = turn_context.activity.recipient.id
 
-        # TODO: find_best_answer should be called on different set based on context
+        question = turn_context.activity.text
+        best_answer, most_similar_question, score = self._find_best_answer(question, conversation_data.context)
+        if score < UNKNOWN_THRESHOLD:
+            best_answer = UNKKNOWN_ANSWER
+        self.bot_sheet.log_answers(question, most_similar_question, best_answer, score)
```

Comment:
How about adding a log entry for the conversation_state? Just serializing it into a json and putting it into a new spreadsheet column would be fine. That would help with debugging.

Reply:
Sure let me do that